network/p2p/validators.go

```diff
@@ -171,7 +133,7 @@ func (v *Validators) Top(ctx context.Context, percentage float64) []ids.NodeID {
 	return top
 }
 
-// Has returns if nodeID is a connected validator
+// Has returns if nodeID is a validator
```

Comment:
This is an invariant change. I believe this doesn't matter because:
1. In the `ValidatorGossiper` we don't care if we are connected to ourselves or not
2. In our `ValidatorHandler` if we are receiving a message over p2p we are connected to the peer

I'm not actually sure why we cared that this peer was connected to begin with... so either I overlooked this in the original implementation or I am forgetting something and this change is introducing a bug.